src/content/plugins/split-chunks-plugin.md

```diff
@@ -283,8 +285,24 @@ module.exports = {
     splitChunks: {
       cacheGroups: {
         vendors: {
+          // example 1: regex that matches `node_modules` and some other source folders.
```

Comment:
Can we please move the comments out of JS code snippet and make these separate examples with proper paragraph outlines? Like here: https://webpack.js.org/configuration/output/#outputfilename